app/lib/features/chat/widgets/bubble_builder.dart

```diff
@@ -331,15 +331,15 @@ class __EmojiContainerState extends ConsumerState<_EmojiContainer>
 
   @override
   Widget build(BuildContext context) {
-    List<String> keys = [];
-    if (widget.message.metadata != null) {
-      if (widget.message.metadata!.containsKey('reactions')) {
-        Map<String, dynamic> reactions = widget.message.metadata!['reactions'];
-        keys = reactions.keys.toList();
-      }
-    }
     return LayoutBuilder(
       builder: (context, constraints) {
+        Map<String, dynamic> reactions = {};
+        List<String> keys = [];
+        final metadata = widget.message.metadata;
+        if (metadata == null) return const SizedBox();
+        if (!metadata.containsKey('reactions')) return const SizedBox();
```

Comment:
let's combine that for compactness, and use the preferred block-format for the if...

```suggestion
        if (metadata == null || !metadata.containsKey('reactions')) {
          return const SizedBox();
        }
```

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1580/commits/86828671a4869455361bf9db46c158a6173842e5